.github/workflows/ci-on-push.yaml

```diff
@@ -1,6 +1,6 @@
 name: Kata Containers CI
 on:
-  pull_request_target:
+  pull_request_target: # zizmor: ignore[dangerous-triggers]
```

Comment:
This trigger is safe today but is still a weak point in our CI as if we by mistake remove some of the mitigations then it will be exposed. I'm saying that because I think we should leave an inline comment to always remember us of that risk. Disable the warning is okay.